### DISCUSSION TITLE
Formatting phone numbers and generating emails

### DISCUSSION THREAD

**Discussion by userjoshs** [faker-js/faker#314] [Q&A] [answered]
Hey guys,

I am using faker for the first time so need some inputs for its usage.
I am using something like this:
 export function newuser(){
    return {
        firstname : faker.name.firstName(),
        lastname : faker.name.lastName(),
     email : faker.internet.email(),
     phone : faker.phone.phoneNumber('04########')
    }
}
I have two questions :
1. Is there any way I could generate email using the first name and last name properties? 
 so if faker generates 
                                       firstname : Johnny
                                       lastname : Bravo 
                               then
                                       email : johnny.bravo@xyz.com
                                     
2. Also is providing '04########' as format a correct way of generating mobile numbers in format starting with 04 followed by 8 digits?(It does work Just want to know if its the right way)

**Comment by ST-DDT** [comment, marked as answer]
1) Yes its possible:
https://github.com/faker-js/faker/blob/3242dfc4b2f566931c3687173f6361a6384b6247/src/internet.ts#L203

2) Yes, that should be fine.
